recipes/example/ispyb/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "ispyb" %}
+{% set version = "4.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "c1f500301de5abfc37f28c88673d5970064d003e73dd5b6a95433292b07adfb3" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - enum34  # [py27]
+    - mysql-connector-python
+
+test:
+  imports:
+    - ispyb
+    - ispyb.connector
+    - ispyb.connector.mysqlsp
+    - ispyb.connector.ws
+    - ispyb.interface
+    - ispyb.legacy
+    - ispyb.legacy.driver
+    - ispyb.legacy.driver.mysql
+    - ispyb.legacy.interface
+    - ispyb.sp
+    - ispyb.ws
+  requires:
+    - mock
```

Comment:
Are these needed for the imports?

Reply:
I don't think so, these were pulled in from the `skeleton` import.  Looking at the code, I think they are only needed for actually running the tests.